timemachine/fe/chiral_utils.py

```diff
@@ -342,3 +347,56 @@ def find_chiral_bonds(mol):
             chiral_bonds.add(tuple(sorted([match[0], match[1]])))
 
     return chiral_bonds
+
+
+def find_canonical_amide_bonds(mol):
```

Comment:
This seems to have a lot of overlap with `find_chiral_bonds` defined above, but not sure if it makes sense to combine them.

Reply:
Let's leave them separate for now. The `find_chiral_bonds` code is probably unused/deprecated.